jetty-server/src/main/config/etc/jetty-https.xml

```diff
@@ -25,5 +25,7 @@
       </New>
     </Arg>
   </Call>
-  
+
+  <Call name="setName"><Arg>https</Arg></Call>
```

Comment:
Also, Default jetty-https.xml should not have a name set.
Perhaps it could have an optional <Property> but not a specific name set.

Reply:
good idea 👍

Reply:
The name for this connector should not be "https", it should be "tls" as it can use alpn to negotiate other protocols.
Again, I don't mind having default names... but ok without also.

Reply:
I've been thinking about it. seems to me like it'd be better to just have these default names? Then people using this new feature wouldn't have to go in and set them manually? 
what is the drawback of having the default names?

Reply:
went back to no defaults. people can add them when they are in my situation easy enough. done.